### PR TITLE
Adarsh-Ovhal-59

### PR DESCRIPTION
**Pull request by Adarshovhal**

### CODE REVIEW

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
change folder name Adarsh_Ovhal_59

File: SubmittedAssignments/Adarsh_Ovhal_59/css/style.css [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+	top: auto;
+	width: 297mm;
+	height: 210mm;
+	overflow: hidden;
+	position: relative;
+}
+	
+.certificate-wrapper .certificate-background .recttop { 
+	position: absolute;
+    width: 250px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Adarsh_Ovhal_59/css/style.css:40]: do not use px

File: SubmittedAssignments/Adarsh_Ovhal_59/css/style.css [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+    height: 202px;
+    top: -132px;
+    background-color:rgb(23, 23, 77);
+    transform: skewY(-46deg);
+}
+
+.certificate-wrapper .certificate-background .rectleft {
+		position: absolute;
+		width: 400px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Adarsh_Ovhal_59/css/style.css:49]: do not use px

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

**Review by Bhosle07 — COMMENTED** [review]
"Check the folder structure to ensure it follows the correct format. In preview.html, verify all classes are present. In style.css, ensure units of measurement are in mm and pt. In index.html, confirm all background classes are included."

**Review by Bhosle07 — COMMENTED** [review]
"Check the folder structure to ensure it follows the correct format. In preview.html, verify all classes are present. In style.css, ensure units of measurement are in mm and pt. In index.html, confirm all background classes are included."

**Pull request merged**